lib/index.js

```diff
@@ -332,19 +332,23 @@ internals.deferPipeUntilSocketConnects = function (req, stream) {
 
     const onSocket = (socket) => {
 
+        if (!socket.connecting) {
+
```

Comment:
No empty line here...

Reply:
I'll fix this up. I'm reviewing and testing before the publish